src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/MyApp1/Program.cs

```diff
@@ -0,0 +1 @@
+Console.WriteLine("Hello, world!");
```

Comment:
Now I understand the draft pr :)

Reply:
Yep, I was just making sure the build works before I bring in a load of content.